opencog/nlp/chatbot-psi/actions.scm

```diff
@@ -72,8 +72,9 @@
 (define (say . words)
     (define utterance (string-join (map cog-name words)))
 
-    ; Remove those '[' and ']' that may exist in the output
-    (set! utterance (string-filter (lambda (c) (not (or (char=? #\[ c) (char=? #\] c)))) utterance))
+    ; Remove those '[', ']' and '\' that may exist in the output
```

Comment:
ooofdah. quotes are hard. Can you add a short comment that says `; The backslash is sometimes generated by AIML rules. The square brackets are sometimes generated by ...`  ?  In the short run, stripping stuff out is OK but In the long run, there should not be garbage in the WordNode string ...